gourmet/gtk_extras/LinkedTextView.py

```diff
@@ -169,17 +178,16 @@ def visibility_notify_event(self, text_view, event):
         self.set_cursor_if_appropriate (text_view, bx, by)
         return False
 
-    def follow_if_link (self, text_view, iter):
-        ''' Looks at all tags covering the position of iter in the text view,
+    def follow_if_link(self, text_view, itr) -> None:
```

Comment:
These changes should not be here, even though the function is broken and should be fixed.

Reply:
This is now fixed. Links of any kinds are now handled correctly :)